docs/wiki/[Examples]-Deploy-Output-Values.md

```diff
@@ -0,0 +1,116 @@
+## Overview
+
+This example code helps you to extract required Terraform output values, which allow you to export structured data about your resources. You can use this data to configure other parts of your infrastructure with automation tools, or as a data source for another Terraform workload deployment folder, Outputs necessary to share data from a root module to child module and vice versa.
+
+Here is example how you can configure output for Azure landing zones architecture using Terraform. 
+
+This is a good starting point to learn how to configuring the landing zone state output, learn how to effectively utilize them in workload terraform modules. 
+
+> IMPORTANT: Ensure the module version is set to the latest, and don't forget to run `terraform init` if upgrading to a later version of the module.
+
+In your Landing Zone example folder create outputs.tf file and copy below terraform code for landing zone state output
+
+### Landing Zone root module folder `outputs.tf` 
+``` hcl
+# The following script provides summary of all resources output.
+# Created by the enterprise_scale module, formatted to allow easy identification of the resource IDs as stored in the Terraform state.
```

Comment:
When referring to the module, we try to use either `caf-enterprise-scale` or `Azure landing zones Terraform module`. Please can you use one of these for consistency?

We have also moved away from referencing "Enterprise-Scale" or "ESLZ", and prefer to use "Azure landing zones" or "ALZ". If you could update these references too, that would be great.